docs/blueprints/EP036.rst

```diff
@@ -52,9 +51,75 @@ Table Group
 -----------
 
 **of_multi_table** will let know other NApp about their determined table. Since it is not necessary to send the entire pipeline, the content of the event **kytos/of_multi_table.enable_table** is obtained from the pipeline key ``napps_table_groups`` value. A value example could be ``{"mef_eline": ["epl"]}`` and it means that **mef_eline** EPL flows should be installed in the table where this value came from.
+To match all the flows, "base" will be use. Additionally we already have defined table groups:
+
+Network protocol _ address
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table:: Example: "ipv6_dst"
+   :widths: 70 30
+   :header-rows: 1
+   :class: center
+
+   * - Network Protocol
+     - Address
+   * - | eth
+       | ipv4
+       | ipv6
+       | tcp
+       | udp
+     - | dst
+       | src
+
+Action
+~~~~~~
+
+.. list-table:: Example: "push_vlan"
+   :width: 150px
+   :header-rows: 1
+   :class: center
+
+   * - Action
+   * - | push_vlan
+       | pop_vlan
+
+VLAN _ groups
+~~~~~~~~~~~~~
+
+.. list-table:: Example: "vlan_vid"
+   :width: 150px
+   :header-rows: 1
+   :class: center
+
+   * - Group
+   * - | vid

```

Comment:
I wonder if we should use in the group the action OF match field like `dl_vlan`, and `dl_vlan_pcp`, I realize it's just an example, but to avoid new readers to get confused, wdyt? or did you mean to convey a different idea?

Reply:
I just noticed these example are not the best. I am going to confirm with @italovalcy for table groups classification. Do you know some that `of_multi_tables` will be using?

Reply:
on e2e tests Italo had confirmed [on this thread](https://github.com/kytos-ng/kytos/pull/359#discussion_r1160149627) that just a basic would suffice though

Reply:
Changed it to flow classification and only `base`, `epl` and `evpl`.